parking_permits_app/apps.py

```diff
@@ -3,4 +3,4 @@
 
 class GraphqlAppConfig(AppConfig):
```

Comment:
For the sake of consistency, consider naming this class to `ParkingPermitsAppConfig`

Reply:
Totally missed this, nice catch, done in 971250cff7b4a498ddef1d66bfde33faa40dcd2b 👍🏼